docs/src/release_notes/v1.20.md

```diff
@@ -0,0 +1,55 @@
+# Release notes for CloudNativePG 1.20
+
+History of user-visible changes in the 1.20 minor release of CloudNativePG.
+
+For a complete list of changes, please refer to the
+[commits](https://github.com/cloudnative-pg/cloudnative-pg/commits/release-1.20)
+on the release branch in GitHub.
+
+## Version 1.20.0
+
+**Release date:** April 27, 2023
+
+!!! Important "Important changes from previous versions"
+    CloudNativePG 1.20 introduces some changes to the default behavior of a
+    few features, compared to previous versions of the operator. The goal of
+    these changes is to improve the resilience of a Postgres cluster out of the
+    box through convention over configuration. For clusters with one or more
+    replicas:
+    
+      - Replication slots for high availability are now enabled by default,
+        unless explicitly disabled
+      - Backup from standby is now enabled by default, unless `target` is
+        explicitly set to `primary`
+      - Restart of the primary is now the default method to complete the
+        unsupervised rolling update procedure (`primaryUpdateMethod`
+        defaults to `restart`, unless explicitly set to `switchover`)
+    
+    Finally, the `-any` service is now disabled by default. For further information,
+    please refer to the ["Installation and upgrades" section](installation_upgrade.md).
+
+Features:
+
+- **Declarative role management:** introduce the `managed.roles` stanza in the
+  `Cluster` spec to provide full lifecycle management of database roles, by
+  providing an abstraction to the related DDL commands in PostgreSQL, such as
+  `CREATE ROLE` and `ALTER ROLE` (#1524, #1793 and #1815)
+- **Declarative hibernation of a PostgreSQL cluster:** introduce a new
+  annotation called `cnpg.io/hibernation` to declaratively hibernate a
+  PostgreSQL cluster by deleting all pods and keeping the PVCs only; the feature
+  also implements the inverse procedure (#1657)
```

Comment:
OK, want to double check this.
On 1.20, declarative hibernation is listed as a feature. In 1.19 and 1.18 as an enhancement.
IMO we should categorize the same way in all three branches, and that should be as feature.
However, I don't have (very) strong feelings here.